src/content/en/fundamentals/engage-and-retain/push-notifications/permission-ux.md

```diff
@@ -0,0 +1,177 @@
+to enable and disable push messaging, without the need of cluttering your
+web app's UI.
+
+A good example of this is [Google I/O's 2016 site](https://events.google.com/io2016/). When you
+first load up the Google I/O site, you aren't asked to do anything,
+the user is left to explore the site.
+
+![When you first load the page, no prompt, just calm on Google IO.](./images/ux-examples/google-io/google-io-first-load.png){: .device-image .center-image }
+
+After a few visits of clicking the menu item on the right, a settings panel is reveal to the
+user, allowing them to set up and manage notifications.
+
+![Settings panel on Google IO's web app for push messaging.](./images/ux-examples/google-io/google-io-settings-panel.png){: .device-image .center-image }
+
+Clicking on the checkbox displays the permission prompt, no hidden surprises.
+
+![Google IO's web app displaying the permission prompt.](./images/ux-examples/google-io/google-io-permission-prompt.png){: .device-image .center-image }
+
+After the permission has been granted the checkbox is checked and the user is
+good to go. The great thing about this UX is that the location to sign up for
+push is the same location to disable push.
+
+
+
+### Passive Approach
+
+One of the easiest ways to offer push to a user is to have a button
+or toggle switch that enables / disables push messages in a location
+on the page that is consistent throughout a site.
+
+This doesn't drive users to enable push notifications, but consistency and allowing users to
+opt in and out easily without constant nudging gives users a way to engage with a site /
+service / brand if and when they want to. For sites like blogs that might have some regular
+viewers as well as high bounce rates, this is a solid option as it targets regular viewers
+without annoying drive-by visitors.
+
+On my personal site I have a toggle switch for push messaging in the footer.
+
+![Example of Gauntface.com push notification toggle in
+footer.](./images/ux-examples/gauntface/gauntface-intro.png)
+
+It's fairly out of the way, but for regular visitors it should get enough
+attention from readers wanting to get updates. People landing on my site to get
+some information and leave are completely unaffected, I doubt they even notice
+the the toggle switch.
+
+If you grant permission the state of the toggle switch changes and remains in the same location
+through the pages.
+
+![Example of Gauntface.com with notifications
+enabled.](./images/ux-examples/gauntface/gauntface-enabled.png)
+
+### The Bad UX
+
+Those are some of the common practices I've noticed on the web. Sadly there is one very common
+bad practice.
+
+The worst thing you can do is instantly show the permission dialog to a user as soon as they
+land on your site.
+
+They have zero context on why they are being asked for a permission, they may
+not even know what your website is for, what it does or what it offers. Blocking
```

Comment:
Change 'Blocking permission...' sentence to 'It's not uncommon for users to block immediate permission requests, especially because they don't know why they're being asked, and the request pop-up is blocking what they're trying to do.